### ISSUE TITLE
Cannot add new contact on Android (on Chromebook)

### ISSUE THREAD

**Issue by tiyb** [oxen-io/session-android#1135]
**Code of conduct**

- [x] I have read and agree to adhere to the [Code of Conduct](https://github.com/oxen-io/session-android/blob/master/CODE_OF_CONDUCT.md).

**Describe the bug**

Using the Android version of Session on a **Chromebook**, there is no ability to add a new contact (or join a community) via Session ID.

**To reproduce**

1. Go to the session main screen, and click the + Button
2. On the **New Conversation** screen tap **New Message** or **Join Community**
3. Paste the Session ID
4. Nothing happens

**Screenshots or logs**

Navigate to **Join Community** screen:

![Screenshot 2023-03-20 11 23 53 AM](https://user-images.githubusercontent.com/47612669/226388357-dada0b9c-6473-45ee-8b4c-6ee6d79789fd.png)

Paste Session ID to Session Community:

![Screenshot 2023-03-20 11 24 29 AM](https://user-images.githubusercontent.com/47612669/226388395-9e3a9513-459b-4375-9979-4ffbe77342a9.png)

Nothing happens; just a text box with the Session ID, but no button to click, no UI to indicate how to move forward.

**Smartphone (please complete the following information):**

 - Device: Google Pixel Slate
 - OS: ChromeOS Version 110.0.5481.181 (Official Build) (64-bit); Android Version 11
 - Version of Session or latest commit hash: Latest from App Store; 1.16.5

**Additional context**

* Also using Proton VPN.
* Have app set to "phone mode" on ChromeOS, but same results when using "tablet" mode

**Comment by tiyb** [comment]
Update: I installed the Linux version of the app on the same Chromebook, and there is a **Next** button in the UI, which becomes enabled when a Session ID is entered.

![Screenshot 2023-03-20 2 00 07 PM](https://user-images.githubusercontent.com/47612669/226429080-0173fa62-3329-43f8-b02b-fa171352204f.png)

**Comment by hjubb** [comment]
Just as a side note, the "join community" page is for joining an open group / community via a URL, the session ID should only be entered in the "New Message", although there should also be confirmation buttons which look like they're being cut off...

**Comment by tiyb** [comment]
Yeah, the point is the lack of a "Next" button when trying to use the Android app on a Chromebook. I used the Community screen because I had a Session ID handy (that I was comfortable to share), but the UI is missing the button in the New Message screen as well.

**Comment by hjubb** [comment]
Thanks for more info. I'm not sure if this is a chromebook specific issue, the UI for that new conversation screen seems different but it might just be due to the resolution I'm not too sure. we can probably move the next button to below the input as opposed to offset from the bottom (might fix it from not displaying properly that way)

**Comment by tiyb** [comment]
Don't know if this is on a roadmap, but it would be great if the change could be made. At this point I have to install the Linux subsystem just to use the app on a Chromebook, which is sub-optimal. 

But it also feels like the kind of visual change that will work properly on any device, once it's made.
